### DISCUSSION TITLE
Subtitle adjustment - Wrong color tokenProposal title

### DISCUSSION THREAD

**Discussion by LucasAlbuquerque0** [Telefonica/mistica-design#1626] [Proposals]
### Goal

Replacing TextSecondary in Subtitle with TextPrimary in components

- BoxedRow
- Row
- BoxedAcoordion
- Accordion
- Snap Card

### Context

It was noticed that some components have the TextSecondary Color Token in the Subtitle. Currently the Subtitle uses TextPrimary and only the Description contains TextSecondary.

### Impact

Maintain consistency and standardization of behavior, ensuring that Subtitle behaves the same in all components.

### Discussion Questions

Just to confirm, is using TextPrimary correct in Subtitle?

### Do you have a visual proposal?

https://www.figma.com/file/42ITbuunFiKBFihDqNgo2s/Subtitle---Text-Secondary?type=design&node-id=631%3A36049&mode=dev&t=Poi0sPFQkMJllKRe-1

**Comment by yceballost** [comment]
Yes! 🙏 There is a discussion talking about the same topic
https://github.com/Telefonica/mistica-design/discussions/848

You can write in #848 to supporting the change!

I will close this discussion to avoid duplicates

**Reply by yceballost** [reply]
@LucasAlbuquerque0 can you open a jira ticket in OBVIVO to prioritize this?

**Reply by LucasAlbuquerque0** [reply]
Of course, I will create the task with the components mapped for the improvement

**Reply by LucasAlbuquerque0** [reply]
@yceballost Link Jira: https://jira.tid.es/browse/OBVIVO-2322
